composer.json

```diff
@@ -6,6 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^8.0.2",
+        "firebase/php-jwt": "^6.3",
```

Comment:
@sabinshrestha8 Specify your reason for using this package.

Reply:
In sanctum, we have to globally put the same expiration time for every token either it is generated  for bearer token or other type of token plus if we use schedular command or like that then we have to specify like after specific days delete token from the database to make it work like something which i have seen in the docs which i may be wrong about and when it comes to Jwt package it is easy to use and we can like specify the expiration time directly which after its validation it will not work .